ember-rails.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.add_dependency "railties", [">= 3.1"]
   s.add_dependency "active_model_serializers"
   s.add_dependency "barber", [">= 0.6.0"]
+  s.add_dependency "es6_module_transpiler-rails"
```

Comment:
Ah, thanks @rwjblue 

In sprockets 4, 6to5 will become  default ES6 transpiler:  https://github.com/sstephenson/sprockets/pull/682 
Should we also use 6to5?